### PR TITLE
New parameters for SmbGlobalMapping

### PR DESCRIPTION
**Pull request by Xelu86**
# PR Summary

New parameters have been added to SmbGlobalMapping for WS25. Update related cmdlets to conform to style guide.

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docset/winserver2025-ps/smbshare/New-SmbGlobalMapping.md [hunk 1]
```diff
@@ -58,8 +70,30 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -BlockNTLM
+
+Specifies whether to block NT LAN Manager (NTLM) authentication. If this parameter is set to
```
Comment (nedpyle) [inline docset/winserver2025-ps/smbshare/New-SmbGlobalMapping.md:75]: differs from previous text used in new-smbshare PR

File: docset/winserver2025-ps/smbshare/New-SmbGlobalMapping.md [hunk 2]
```diff
@@ -148,8 +207,42 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -QuicPort
+
+Specifies the port number to be used for Quick UDP Internet Connections (QUIC) protocol
```
Comment (nedpyle) [inline docset/winserver2025-ps/smbshare/New-SmbGlobalMapping.md:212]: QUIC, not an acronym

**Review by nedpyle — CHANGES_REQUESTED** [review]
A few small changes

File: docset/winserver2025-ps/smbshare/New-SmbGlobalMapping.md (hunk 1 above)
Comment (Xelu86) [reply docset/winserver2025-ps/smbshare/New-SmbGlobalMapping.md:75]: New-SmbMapping PR

**Pull request merged**